### PR TITLE
Package yosqlite.0.1

### PR DESCRIPTION
**Pull request by finalclass**
### `yosqlite.0.1`
Simple Sqlite3. Yojson+Sqlite3 amalgamation
Convenience functions for working with Sqlite3 library

---
* Homepage: https://github.com/finalclass/yosqlite
* Source repo: git+https://github.com/finalclass/yosqlite.git
* Bug tracker: https://github.com/finalclass/yosqlite/issues

---
:camel: Pull-request generated by opam-publish v2.3.1

### CODE REVIEW

**Comment by finalclass** [comment]
1. From what I see the package does not build because of Sqlite3 is not building on windows.
2. Regarding the linter, the name "yosqlite" is really far away from "ezsqlite". I don't see how anyone would confuse these two. Isn't it too strict?

**Comment by shonfeder** [comment]
> 2\. Regarding the linter, the name "yosqlite" is really far away from "ezsqlite". I don't see how anyone would confuse these two. Isn't it too strict?

Yes, 100%. Sorry about that.

File: packages/yosqlite/yosqlite.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Simple Sqlite3. Yojson+Sqlite3 amalgamation"
```
Comment (shonfeder) [inline packages/yosqlite/yosqlite.0.1/opam:3]: I'm not totally clear how to parse this synopsis. Is the `.` meant to behere?

File: packages/yosqlite/yosqlite.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+description: "Convenience functions for working with Sqlite3 library"
+maintainer: ["Szymon Wygnański"]
```
Comment (shonfeder) [inline packages/yosqlite/yosqlite.0.1/opam:5]: Could you provide some way of contacting you here please? An email address is our usual way.

**Review by shonfeder — COMMENTED** [review]
Thanks for publishing this! Looks likes like a nice simple API :)

The conf-sqlite package is indeed the problem on windows:

```
  #=== ERROR while compiling conf-sqlite3.1 =====================================#
  "pkg-config": command not found.
  
  
  <><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
  +- The following actions failed
  | - fetch base         v0.17.1
  | - fetch cppo         1.6.9
  | - fetch csexp        1.5.2
  | - build conf-sqlite3 1
  +- 
  +- The following changes have been performed (the rest was aborted)
  | - install conf-pkg-config 3
  +- 
  # Run (& opam env) -split '\r?\n' | ForEach-Object { Invoke-Expression $_ } to update the current shell environment
  
  The former state can be restored with:
      D:\opam\bin\opam.exe switch import "D:\\opamroot\\default\\.opam-switch\\backup\\state-20240730203020.export"
yosqlite.0.1 failed to build.

Exception: D:\a\_temp\2e137196-066e-4f1c-9404-f804f58ca2a8.ps1:19
```

Just two nits on the package data. Thanks for looking at them!

File: packages/yosqlite/yosqlite.0.1/opam (hunk 1 above)
Comment (shonfeder) [reply packages/yosqlite/yosqlite.0.1/opam:3]: No need to change this if it may be a reading comprehension issue on my part, btw. :)

**Comment by finalclass** [comment]
@shonfeder I've applied the review suggestions. Could you recheck?

File: packages/yosqlite/yosqlite.0.1/opam (hunk 1 above)
Comment (finalclass) [reply packages/yosqlite/yosqlite.0.1/opam]: Yep, it might be confusing. I'm changing the synopsis and description.

**Comment by mseri** [comment]
Thanks

**Pull request merged**